R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -196,6 +196,12 @@ test_that("create DataFrame from RDD", {
   expect_equal(dtypes(df), list(c("name", "string"), c("age", "int"), c("height", "float")))
   expect_equal(as.list(collect(where(df, df$name == "John"))),
                list(name = "John", age = 19L, height = 176.5))
+  expect_equal(getNumPartitions(toRDD(df)), 1)
```

Comment:
it looks like it is intentional:
https://github.com/apache/spark/blob/master/R/pkg/R/context.R#L115